juriscraper/opinions/united_states/state/mass.py

```diff
@@ -25,9 +25,19 @@
 class Site(OpinionSiteLinear):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.url = "https://www.mass.gov/service-details/new-opinions"
+        self.url = "https://www.mass.gov/info-details/new-opinions"
         self.court_id = self.__module__
         self.court_identifier = "SJC"
+        self.headers = {
+            "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
+            "Sec-Fetch-Site": "none",
+            "Sec-Fetch-Mode": "navigate",
+            "Host": "www.mass.gov",
+            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.3 Safari/605.1.15",
+            "Accept-Language": "en-US,en;q=0.9",
+            "Sec-Fetch-Dest": "document",
+            "Connection": "keep-alive",
+        }
```

Comment:
I think this won't be used by AbstractSite. You would have to modify `self.request["headers"]`
When changing the headers, I no longer get a 403, but the scraper is not collecting any cases. Maybe the HTML changed?

Reply:
this is meant to be used with CL 

similar to the Minnesota scraper - it stores it in the site.headers to be used

Reply:
I've updated the changes you suggested.

Reply:
`mass` is working now

Reply:
thanks @grossir